plugins/php/php_fpm_process

```diff
@@ -175,10 +246,31 @@ if($body =~ m/total processes: (.*?)\n/) {
         $TOTAL = $1;
   print "total.value ".$TOTAL."\n";
 }
+if($body =~ m/max active processes: (.*?)\n/) {
+        $MAX = $1;
```

Comment:
Did you mix whitespace here?
(and in some occurrences below)

Reply:
Yes and no. I tried to match the (weird) indent that was already there. I'm more than happy to fix it completely.